F/FLINT/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FLINT"
+version = v"0.0.1"
+
+# Collection of sources required to build FLINT
+sources = [
+    GitSource("https://github.com/wbhart/flint2.git","dd1021a6cbaca75d94e6e066c26a3a5622884a7c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd flint2/
+
+libsubdir=lib
+
+if [[ ${target} == *musl* ]]; then
+   # because of some ordering issue with pthread.h and sched.h includes
+   export CFLAGS=-D_GNU_SOURCE=1
+   # and properly define _GNU_SOURCE here as well to avoid many warnings
+   sed -i -e 's/#define _GNU_SOURCE$/#define _GNU_SOURCE 1/' thread_pool.h configure
+elif [[ ${target} == *mingw* ]]; then
+   # fix arch detection:
+   sed -i -e 's/$(ARCH)/$ARCH/g' configure
+   extraflags=--reentrant
+   libsubdir=bin
+fi
+
+./configure --prefix=$prefix --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix ${extraflags}
+make -j${nproc}
+make install LIBDIR=$libsubdir
```

Comment:
We already have the [environment variable](https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1) for this, no need to manually set it (remove also above)
```suggestion
make install LIBDIR=${libdir}
```

Reply:
`$libdir` is not correct here as the Makefile expects this to be just the `lib` or `bin` but `$(basename ${libdir})` should do and is nicer than setting it manually.